woocommerce/woocommerce
# Add `wc com connect` command

### All Submissions:






### Changes proposed in this Pull Request:

PR adds `wc com connect` command which allows stores to be connected to WCCOM via CLI script

Closes https://github.com/Automattic/woocommerce.com/issues/13888.

### How to test the changes in this Pull Request:

1. Generate an application password following instructions from https://github.com/Automattic/woocommerce.com/pull/13974. Save the password for later use
2. From your tes site, be sure you're connecting to a local instance of 'woocommerce.com' using `woocommerce_helper_api_base` filter
```
add_filter( 'woocommerce_helper_api_base', function () {
	return 'https://woocommerce.test/wp-json/helper/1.0';
} );
```
3. From the terminal, run: `wc com connect`
4. When asked, past the application password from step 1.
<img width="222" alt="Screen Shot 2022-07-27 at 04 15 26" src="https://user-images.githubusercontent.com/532402/181223222-6266eb7d-8342-4672-b27b-2a30576a67c6.png">

5. Site should be connected now.
7. Run `wc com connect` again, see an error message when the site is already connected
7. Run `wc com connect --force`, check that site is disconnected and connected again
7. Run `wc com connect --password=APPLICATION_PASSWORD --force`, password shouldn't be prompt 

### Other information:




### FOR PR REVIEWER ONLY:
